internal/inspect/inspect.go

```diff
@@ -34,30 +35,40 @@ func Inspect(out io.Writer, app *types.App, argParameters map[string]string, ima
 
 	// Add Service section
 	printSection(out, len(config.Services), func(w io.Writer) {
+		sort.Slice(config.Services, func(i, j int) bool {
+			return config.Services[i].Name < config.Services[j].Name
+		})
 		for _, service := range config.Services {
 			fmt.Fprintf(w, "%s\t%d\t%s\t%s\n", service.Name, getReplicas(service), getPorts(service.Ports), service.Image)
 		}
 	}, "Service", "Replicas", "Ports", "Image")
 
-	// Add Network section
-	printSection(out, len(config.Networks), func(w io.Writer) {
-		for name := range config.Networks {
+	outputSortedStringKeys := func(m interface{}, w io.Writer) {
+		v := reflect.ValueOf(m)
```

Comment:
holly shit, we need generics!

Reply:
Can we change things to avoid reflection?
Something like introducing a interface 
```
type sectionItem interface{
  print(w tabwriter.Writer) error
} 
```
and working with a typed `map[string]sectionItem` ?

Reply:
I think (although I've not tried it yet) that you'd end up reimplementing `func (x *type) print()` N times for the N variants, at which point you've lost the benefit of taking away the code duplication since you repeat yourself as much as before.

IOW we might as well drop the final commit here since it results in approximately/morally the same thing.

@simonferquel shall I just drop the final commit?